appveyor.yml

```diff
@@ -5,9 +5,8 @@ environment:
   - TARGET: x86_64-pc-windows-msvc
     ALLOW_PR: 1
   - TARGET: i686-pc-windows-msvc
```

Comment:
No artifacts are being produced for MSI currently, see https://github.com/rust-lang-nursery/rustup.rs/blob/ac5bffad7ba126b2313692c3dfce83ec91cb7fea/ci/prepare-deploy-appveyor.ps1#L9-L12

Changing that situation was part of #661, but I didn't include it here. Will do in another PR.